src/core/pointcloud/qgspointcloudrendererregistry.cpp

```diff
@@ -188,3 +190,21 @@ QgsPointCloudRenderer *QgsPointCloudRendererRegistry::defaultRenderer( const Qgs
   return renderer.release();
 }
 
+QgsPointCloudCategoryList QgsPointCloudRendererRegistry::defaultCategories( const QgsPointCloudLayer *layer )
```

Comment:
I would suggest to rename this to make it clear it's meant only for "Classification" attribute... e.g. defaultClassificationAttributeCategories ? (maybe that's too long :-) )

Reply:
I realized today this is a bit misleading too, since what it does is get all available classification classes, it just uses the default colors/names for the default ones.
So how about `classificationAttributeCategories`?

Reply:
`classificationAttributeCategories` sounds good :+1: